### DISCUSSION TITLE
🇩🇪 Zak McKracken and the Alien Mindbenders

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/eltikia.github.io#2] [Q&A] [answered]
Was bedeute die Beschreibungen unter die Videos?

**Comment by Stagyrite** [comment, marked as answer]
Diese Beschreibungen können die Spielen nicht erklären.

### [Zak McKracken and the Alien Mindbenders](https://www.youtube.com/watch?v=S3ETogTxtnY "🇩🇪 Zak McKracken and the Alien Mindbenders - YouTube")

> Zachary hatte keine Ahnung, dass die Telefonrechnung fällig waren. Wie wird er bezahlen? Vergiss nicht zu der Erde retten, Zachary.

Es gib ein Beispielsatze.

> Werner hatte keine Ahnung, dass die Steuern fällig waren.

Es gibt auch ein Trick zu der Telefonrechnung zahlen. Zachary telefoniert und dann zerstört der Zahlung. Mann muss zuerst der Telefonnummer erkannt.

Zachary rettet der Erde und das ist klar für die Spieler.
